cmake/modules/MinGWCrossCompile.cmake

```diff
@@ -84,7 +84,7 @@ IF(NOT DEFINED ENV{MINGW_DEBUG_INFO})
 	MESSAGE("* CMAKE_CXX_COMPILER                   : ${CMAKE_CXX_COMPILER}")
 	MESSAGE("* CMAKE_RC_COMPILER                    : ${CMAKE_RC_COMPILER}")
 	MESSAGE("* WINDRES                              : ${WINDRES}")
-	MESSAGE("* ENV{PKG_CONFIG}                      : $ENV{PKG_CONFIG}")
+	MESSAGE("* PKG_CONFIG_EXECUTABLE                : ${PKG_CONFIG_EXECUTABLE}")
```

Comment:
Can you confirm this was available in [CMake `2.8.9`](https://github.com/LMMS/lmms/blob/master/CMakeLists.txt#L1) ?